lib/partitioned/partitioned_base/redshift_sql_adapter.rb

```diff
@@ -222,7 +222,7 @@ def unique_index_name(name, *partition_key_values)
       #
       def add_references_to_partition_table(*partition_key_values)
         configurator.foreign_keys(*partition_key_values).each do |foreign_key|
-          add_foreign_key(partition_table_name(*partition_key_values),
+          add_foreign_key_constraint(partition_table_name(*partition_key_values),
```

Comment:
Style comment: Would you mind fixing the indents of rest of the arguments being passed in in the next few lines?